shared/main.lua

```diff
@@ -41,6 +41,38 @@ function QBShared.Round(value, numDecimalPlaces)
     return math.floor((value * power) + 0.5) / (power)
 end
 
+-- Debugging purposes. Prints out a supplied table in the server/f8 console.
+
+function QBShared.DumpTable(tbl, nb)
```

Comment:
No need for this, you have `QBCore.Debug` which prints a formatted table already

Reply:
Removed :)